libs/types/src/cdf/election-results-reporting/convert.ts

```diff
@@ -117,10 +132,10 @@ function convertToYesNoContest(
   const undervotes = otherCounts?.Undervotes || 0;
 
   return {
-    contestId: contest['@id'],
+    contestId: trimVxIdPrefix(contest['@id']),
     contestType: 'yesno',
-    yesOptionId: yesOption['@id'],
-    noOptionId: noOption['@id'],
+    yesOptionId: trimVxIdPrefix(yesOption['@id']),
+    noOptionId: trimVxIdPrefix(noOption['@id']),
```

Comment:
I wrestled with whether I should include getters that wrap this pattern but in the end felt like it was overwrought. Open to feedback if this feels dangerous though.